src/main/java/frc/robot/Constants.java

```diff
@@ -16,21 +16,26 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-        // ADD THE PORT NUMBERS TO CONSTANTS
 
         public static final class DriveConstants {
             public static final int kLeftDriveMotor1Port = 0;
             public static final int kLeftDriveMotor2Port = 15;
             public static final int kRightDriveMotor1Port = 3;
             public static final int kRightDriveMotor2Port = 12;
 
-            // pistons that change the gear of the drive train
+            // pistons that shift the gear of the drive train 
             public static final int kDriveSolenoid = 7;
     
-            public static final double kWheelDiameter = 0; // ADD UNIT OF MEASUREMENT
+            public static final double kWheelDiameter = 6; 
 
+            //constant for the distance that mechanism travels for every pulse of the encoder
             public static final double kEncoderDistancePerPulse = 0; // ADD CONVERSION
+            // constant for the speed of the motor per pulse of the encoder
             public static final double kEncoderSpeedPerPulse = 0; // ADD CONVERSION
```

Comment:
We should be able to add these now, since we know the wheel diameter and gearing.  There should be two sets of speed and encoder conversions, one for high gear and one for low gear.